README.md

```diff
@@ -0,0 +1,36 @@
+# Student Help Guide.
+
+This repo contains the source code for a website containing useful content for interns at Red Hat.
+
+This is the github repo for building the site. To view the site go to http://feedhenry.org/student-help-guide/
+
+
+## Folder Structure
+
+/content -> contains the md content for the site - make your changes here
```

Comment:
`contains any assets (images, documents, etc.) and contents written in markdown` wdyt?